calico/getting-started/kubernetes/vpp/getting-started.md

```diff
@@ -124,89 +124,88 @@ Before you get started, make sure you have downloaded and configured the {% incl
 
 
 %>
+
 <label:Install on EKS with DPDK>
 <%
 
 ### Install Calico with the VPP dataplane on an EKS cluster with the DPDK driver
 
 #### Requirements
 
-These instructions require that `eksctl` (>= 0.51) and the `aws` (version 2) CLI are installed on your system to provision the cluster.
-
+DPDK provides better performance compared to the standard install but it requires some additional customisations (hugepages, for instance) in the EKS worker instances. We have a bash script, `init_eks.sh`, which takes care of applying the required customizations and we make use of the `preBootstrapCommands` property of `eksctl` {% include open-new-window.html text='configuration file' url='https://eksctl.io/usage/schema' %} to execute the script during the worker node creation. These instructions require the latest version of `eksctl`.
 
-#### Provision the cluster and configure it for DPDK
 
-DPDK provides better performance compared to the standard install but it requires some additional customisations (hugepages, for instance) in the EKS worker instances. We have created a bash script, `create_eks_cluster.sh`, which automates the whole process right from customising the EKS worker instances (using cloud-init) to creating the cluster and the worker nodegroup. The script has been tested on MacOS and Linux.
+#### Provision the cluster
 
-1. Download the helper script
+1. First, create an Amazon EKS cluster without any nodes.
```

Comment:
Maybe first step is to download eksctl? Or a pre-requisite perhaps?

Reply:
Its mentioned under **Requirements** section at the very top.

Reply:
Isn't that the Line 134 that has been removed in this PR?